src/components/OSCALImplementedComponents.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { styled } from "@mui/material/styles";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableRow from "@mui/material/TableRow";
+
+const ImplementedComponentsTable = styled(TableContainer)(
+  ({ theme }) => `
+  margin-top: ${theme.spacing(2)};
+  display: flex;
+  flex-direction: column;
+  text-transform: capitalize;
+`
+);
+
+export default function OSCALImplementedComponents(props) {
```

Comment:
Again I do not think that this is truly an encapsulation of `OSCALImplementedComponents`. It is `ImplementedComponents` in terms of what it means in `SystemImplementation`.

Reply:
I was thinking this would be reusable in a different context in the Assessment Layer, but it seems like `implemented-components` all fall underneath `local-definitions/inventory-items`. I think this can just be brought into `OSCALInventoryItems`.